test/tests/pbs_smoketest.py

```diff
@@ -1458,3 +1458,20 @@ def test_interactive_job(self):
         self.server.expect(JOB, {'job_state': 'R'}, id=jid)
         self.server.delete(jid)
         self.server.expect(JOB, 'queue', op=UNSET, id=jid)
+
+    def test_man_pages(self):
+        """
+        Test basic functionality of man pages
+        """
+        pbs_conf = self.du.parse_pbs_config(self.server.shortname)
+        manpath = os.path.join(pbs_conf['PBS_EXEC'], "share", "man")
+        pbs_cmnds = ["pbsnodes", "qsub"]
```

Comment:
Why only two commands?  It's a for loop, you might as well toss a couple more in.  Maybe a daemon and qmgr?